### DISCUSSION TITLE
accesso con duckdns in Hassio

### DISCUSSION THREAD

**Discussion by Dregi56** [indomus/forum#737] [HUB: Home Assistant]
Da remoto non riesco più ad accedere con https//nomedominio.duckdns.org:porta
L'integrazione per la scadenza del certificato mi dà il seguente messaggio:
Nuovo tentativo di configurazione: [Errno 113] Host is unreachable
In Lovelance il sensore per la scadenza del certificato, mi dà -18.887 giorni

E' il server duck.org che non funziona, oppure è il certificato che non si è rinnovato attraverso l'add-on?

**Comment by snakuzzo** [comment]
Controlla che l'ip pubblico che hai attualmente sia quello associato al servizio duckdns

**Reply by Dregi56** [reply]
Si. E' quello, è sempre lo stesso, in realtà ho un IP fisso.

**Reply by Dregi56** [reply]
Ho provato a reinstallare l'add-on DuckDNS. Ho provato a reinstallare l'integrazione per il sensore "scadenza certificato" e quando inserisco l' host (https://miodominio.duckdns.org ) mi dice: che il dominio non può essere risolto...la porta aperta è corretta. Non so più che fare.... e se provassi a cancellare da dir SSL i due file fullchain.pem e privkey.pem?

**Reply by Dregi56** [reply]
Oggi è una bella giornata! Ho risolto, era un problema di instradamento porte. Io uso un doppio passaggio per l'istradamento delle porte, modem provider-mio router un aggiornamento del firmware del router mi ha sballato la configurazione di rete verso Internet.
Grazie comunque per aver risposto.
